packages/elements/src/slider/elements/slider.ts

```diff
@@ -424,6 +428,16 @@ export class Slider extends ControlElement {
     this.prepareSliderTrack();
   }
 
+  /**
+   * Perform asynchronous update
+   * @returns promise
+   */
+  protected override async performUpdate(): Promise<void> {
+    this.toAriaLabel = await this.labelTPromise(SliderDataName.to.toUpperCase());
+    this.fromAriaLabel = await this.labelTPromise(SliderDataName.from.toUpperCase());
```

Comment:
let them run independently for better performance.
```suggestion
    [this.toAriaLabel, this.fromAriaLabel] = await Promise.all([
      this.labelTPromise(SliderDataName.to.toUpperCase()),
      this.labelTPromise(SliderDataName.from.toUpperCase())
    ]);
```